src/tests/functional/plugin/shared/src/low_precision_transformations/reduce_max_transformation.cpp

```diff
@@ -31,12 +30,15 @@ std::string ReduceMaxTransformation::getTestCaseName(const testing::TestParamInf
 }
 
 void ReduceMaxTransformation::SetUp() {
-    ngraph::element::Type netPrecision;
-    ngraph::PartialShape inputShape;
+    abs_threshold = 1.1;
```

Comment:
it looks like a big threshold. could you elaborate what was the original threshold for this test before refactoring.